up-l3/utwin/v2/README.adoc

```diff
@@ -118,23 +118,25 @@ uTwin will initialize its cache on startup with the following scenarios:
 * when running without persistent storage, the cache will be initialized to an empty condition
 * when running with persistent storage (future option), the cache can be initialized to the last saved state for each Topic.
 
-uTwin MUST indicate when its capacity is exhausted and it is unable to accept a new / updated SetLastValue command.
+uTwin *MUST* indicate when its capacity is exhausted and it is unable to accept a new / updated SetLastEvent command.
 
-uTwin should actively manage its cache to ensure that the oldest and lowest priority Topics are removed from the cache in order to ensure that new/updated content can continue to be accepted.  This should be done as a background/idle task to decouple it from impacting active uTwin transactions, potentially based upon a set threshold (e.g. 95% full).  Criteria for protection of higher priority content will need to be established, along with determination of "staleness" for specific topics/topic classes.
+In the event that limited cache capacity is available for uTwin and the potential exists for it to run out of capacity if all or a significant portion of available Topics provided by or used by on-device uEs, uTwin *SHOULD* actively manage its cache to ensure that the oldest and lowest priority Topics (subject to implementation-specific constraints) are removed from the cache in order to ensure that new content can continue to be accepted.  
+
+If uTwin cache scrubbing is supported, this *MUST* be implemented so as to prevent it from adding latency to active uTwin SetLastEvent transactions, potentially based upon a configurable threshold (e.g. 95% full).  Criteria for protection of higher priority content will need to be established, along with determination of "staleness" for specific topics/topic classes.
```

Comment:
This sentence doesn't read correctly, can you please reword it. You say if then you add MUST so is a a MAY or MUST?